File: src/app/Models/Attendance.ts
export class Attendance{
    public attendanceDate : Date;
    public totalTime : number=1.5;
    public dayName : string="";
    constructor(attendanceDate:Date, totalTime: number , dayname : string){
        this.attendanceDate = attendanceDate;
        this.totalTime = totalTime;
        this.dayName = dayname;
    }
}

File: AttendenceBackEnd/AttendenceBackEnd/Models/Attendance.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace AttendenceBackEnd.Models
{
    public class Attendance
    {

        public int Id { get; set; }
        [Required]
        public DateTime AttendanceDate { get; set; }
        [Required]
        public double TotalTime { get; set; }
        [Required]
        public int AppUserId { get; set; }
        public AppUser AppUser { get; set; }
    }
}
